tests/foreman/api/test_config_template.py

```diff
@@ -0,0 +1,32 @@
+"""Unit tests for the ``config_templates`` paths.
+
+A full API reference is available here:
+http://theforeman.org/api/apidoc/v2/config_templates.html
+
+"""
+from nailgun import client
+from robottelo.common.decorators import skip_if_bug_open
+from robottelo.common.helpers import get_server_credentials
+from robottelo import entities
+from robottelo.test import APITestCase
+
+
+class ConfigTemplateTestCase(APITestCase):
+    """Tests for config templates."""
+
+    @skip_if_bug_open('bugzilla', 1202564)
+    def test_build_pxe_default(self):
+        """@Test: Call the "build_pxe_default" path.
+
+        @Assert: The response is a JSON payload.
```

Comment:
I'd like to be able to care about the response, but that's not a good thing to
do. The response is prose, and prose is unreliable. From the bug report:

> Actual results:
> 
>     'PXE Default file has been deployed to all Smart Proxies'
> 
> Expected results:
> The payload is valid JSON. For example, something like this:
> 
>     {
>         'status': 'success',
>         'message': 'PXE Default file has been deployed to all Smart Proxies',
>     }